### ISSUE TITLE
Inline elements don't parse across newlines

### ISSUE THREAD

**Issue by duckwork** [miyuchina/mistletoe#30]
When parsing Markdown, mistletoe doesn't parse inlines across line boundaries.  Example source:

```markdown
here is a file with *emph* and also
it will have *emph across
lines.*  uh-oh. Let's try **strong** and
also, of course, **strong across
lines.** uh-oh-uh-oh.
```

And example output:

```html
<p>here is a file with <em>emph</em> and also
it will have *emph across
lines.*  uh-oh. Let&#x27;s try <strong>strong</strong> and
also, of course, **strong across
lines.** uh-oh-uh-oh.
</p>
```

Most parsers recognize inlines over newlines.  It'd be nice if mistletoe did, too.  Would it be as simple as setting a `re` flag?

**Comment by miyuchina** [comment]
Yeah, I added `re.DOTALL` flags across all the builtin token types. Some regexes also need to be changed, and there might be some performance implications, but CommonMark does it, so I'm keeping this feature.

Let me know if there's anything weird; I'll do a minor version release to pypi later.

**Comment by duckwork** [comment]
Cool, thanks! I'm looking forward to the pypi release.

**Comment by miyuchina** [comment]
Just released it! Should be working in 0.5.4.
